docs/bbs/AbiPattern.md

```diff
@@ -0,0 +1,23 @@
+# ABIPATTERN Format
+
+AbiPattern stands for *Ability Pattern*.
+
+This file stores a list of abilities. It's use is unknown, but it seems to be related to command melding.
+
+This file is contained within `Menu/Camp.arc`
+
+
+## Header
+| Offset | Type  | Description
```

Comment:
Leave a line between headers and everything else.
Github is known to break formatting otherwise.

Reply:
Okay, it should be fixed now!